src/main/java/org/example/c01/gettingStarted/Notes.txt

```diff
@@ -7,6 +7,13 @@ Statement is complete command to be executed
 Keyword is any one of a number of reserved word, that have a predefined meaning in Java
```

Comment:
You can consider Notes.txt file a application resource file. Move it to src/main/resources folder. 
Here you can read about Maven project standard directory layout
https://maven.apache.org/guides/introduction/introduction-to-the-standard-directory-layout.html

Reply:
done